k8s/zero/deployment/env.yaml

```diff
@@ -19,6 +19,10 @@ spec:
                 fieldRef:
                   apiVersion: v1
                   fieldPath: metadata.namespace
+            - name: BOOTSTRAP_CONFIG_FILE
+              value: "/var/run/secrets/pomerium/bootstrap.dat"
+            - name: BOOTSTRAP_CONFIG_WRITEBACK_URI
+              value: "secret://pomerium-zero/pomerium/bootstrap"
```

Comment:
```suggestion
              value: "secret://$(POMERIUM_NAMESPACE)/pomerium/bootstrap.dat"
```

Reply:
Do you think we should use the same `pomerium` secret for this, or a different one?

Reply:
I wasn't sure if envs would expand here, let me double-check

Reply:
> Do you think we should use the same `pomerium` secret for this, or a different one?

it seemed to make sense: the user has to create that secret with their PZ token, until its created PZ is not starting, and it is kept outside of installation manifests so that when you upgrade it doesn't get overwritten.

Reply:
That should work, but a possible complication is that we have to mark the whole secret with `optional: true` since the `bootstrap.dat` key won't exist on the first run. However, we still want the secret to exist to make sure the zero token will be present.

Reply:
the volume mount is now optional, so the secret doesn't need to have this property anymore; we reference the `POMERIUM_ZERO_TOKEN` env , so if the secret does not exist it would still wait until the user creates this secret.